scripts/timescaledb/before-create.sql

```diff
@@ -0,0 +1,24 @@
+-- Check if one of the timescaledb schemas are present in the database, while the extension is missing
+-- Note: this list assumes any new schemas added to the extension should be present here as well.
+DO $$
+BEGIN
+    IF EXISTS(
```

Comment:
Actually, since this is called before the CREATE EXTENSION command, need to check that timescaledb is not yet installed.